ush/ufsda/archive.py

```diff
@@ -0,0 +1,62 @@
+from r2d2 import store
+from solo.configuration import Configuration
+from solo.nice_dict import NiceDict
+import os
+import shutil
+import datetime as dt
+import ufsda
+
+__all__ = ['atm_diags']
+
+
+def atm_diags(config):
+    # fetch atm analysis obs
+    r2d2_config = {
+        'start': config['prev_valid_time'],
+        'end': config['prev_valid_time'],
+        'step': config['atm_window_length'],
+        'dump': 'gdas',
+        'experiment': config['experiment'],  # change this here and other places to be oper_{dump}
```

Comment:
Comment here I don't think is valid, we would always want the diags to be archived as `experiment`. The `dump` part above should probably be in the config though (not necessary for now)